stl/inc/yvals_core.h

```diff
@@ -1359,7 +1360,11 @@
 #define __cpp_lib_polymorphic_allocator   201902L
 
 #if defined(__cpp_lib_concepts) // TRANSITION, GH-395
+#if _HAS_CXX23
+#define __cpp_lib_ranges 202202L
+#elif _HAS_CXX20 // ^^^ _HAS_CXX23 / _HAS_CXX20 vvv
 #define __cpp_lib_ranges 202110L
+#endif // _HAS_CXX23
 #endif // defined(__cpp_lib_concepts)

```

Comment:
Since the value of `__cpp_lib_ranges` now depends on the standard mode, I believe that the definition should be moved to below

https://github.com/microsoft/STL/blob/bf3ac95af7438f818eac1754c9f16cfdb416cd23/stl/inc/yvals_core.h#L1417

(Specifically, between `__cpp_lib_optional` and `__cpp_lib_shared_ptr_arrays`.)

Reply:
Thanks for pointing that out 👍